scripts/upgrade-facets.js

```diff
@@ -205,7 +205,13 @@ async function main(env, facetConfig) {
     contracts = contracts.filter((i) => i.name !== newFacet.name);
 
     const newFacetInterfaceId = interfaceIdFromFacetName(newFacet.name);
-    deploymentComplete(newFacet.name, newFacet.contract.address, [], newFacetInterfaceId, contracts);
+    deploymentComplete(
+      newFacet.name,
+      newFacet.contract.address,
+      newFacet.constructorArguments,
```

Comment:
```suggestion
      newFacet.constructorArgs,
```

Variable in `scripts/util/deploy-protocol-handler-facets.js` was renamed, so it must be here as well.

Reply:
fixed